pkg/shell/superuser.jsx

```diff
@@ -78,21 +81,28 @@ const UnlockDialog = ({ proxy, host }) => {
             did_prompt = true;
         };
 
+        const maybe_remember = () => {
+            if (rememberAccessLevelOnUnlock) {
+                if (key)
+                    window.localStorage.setItem(key, method);
+            }
```

Comment:
These 4 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17177-20220927-070244-b0e468e8-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)